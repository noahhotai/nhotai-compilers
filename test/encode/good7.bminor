"cof\v\t\0x22"