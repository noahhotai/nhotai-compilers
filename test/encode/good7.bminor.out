Decoded line: cof	"
Encoded line: "cof\v\t\""
